scripts/system/create/entityProperties/html/js/entityProperties.js

```diff
@@ -1842,9 +1860,15 @@ function resetServerScriptStatus() {
 function showGroupsForType(type) {
     if (type === "Box" || type === "Sphere") {
         showGroupsForTypes(["Shape"]);
+        showOnTheSamePage("Shape");
```

Comment:
Should this be the following?
```suggestion
        showOnTheSamePage(["Shape"]);
```
The `showOnTheSamePage()` function expects an array?

Reply:
Changes for this will be committed in PR #215 related to properties changes to avoid regression.